oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+account-analytic
```

Comment:
This is not needed anymore.

If this depends on another pull request, the good method is this one: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

Either, the addon in account analytic will be found by pip automatically

Reply:
Thank you. I deleted this file.